.github/actions/php-test-symfony.yml

```diff
@@ -0,0 +1,43 @@
+name: PHPUnit tests with Symfony
```

Comment:
@nikophil, to clarify, this can be called from _outside_ this repo with:

```yaml
uses: zenstruck/.github/actions/php-test-symfony@main
```

Reply:
lgtm